download.html

```diff
@@ -9,13 +9,8 @@
 
   <div class="col-sm-6">
     <h3>Download Types</h3>
-
-    <p>The prefered way to download NUnit is through <a href="https://www.nuget.org/packages/NUnit/">NuGet</a> package manager.</p>
-
-    <p>NUnit Xamarin Runners are also available through <a href="https://www.nuget.org/packages/nunit.xamarin/">NuGet</a> or
-      from <a href="https://github.com/nunit/nunit.xamarin/releases">GitHub</a>.
-
-      <p>The latest releases of NUnit 3 can always be found on the <a href="https://github.com/nunit/nunit/releases">GitHub releases pages</a>.</p>
+    <p>The preferred way to download NUnit is through <a href="https://www.nuget.org/">NuGet</a> package manager.</p>
```

Comment:
(update to the original text)

```suggestion
    <p>The preferred way to download NUnit is through the <a href="https://www.nuget.org/">NuGet</a> package manager.</p>
```